lib/test_suite/bird_count.ex

```diff
@@ -0,0 +1,53 @@
+defmodule ElixirAnalyzer.TestSuite.BirdCount do
+  @dialyzer generated: true
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise Bird Count
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+  alias ElixirAnalyzer.Constants
+
+  assert_no_call "does not call any Enum functions" do
+    type :essential
+    called_fn module: Enum, name: :_
+    comment Constants.bird_count_use_recursion()
```

Comment:
@neenjaw This doesn't work. I get the error about comment missing because it's:
```
[{{:., [line: 13], [{:__aliases__, [line: 13], [:Constants]}, :bird_count_use_recursion]}, [line: 13], []}]
```

And not a binary, so it doesn't get added (fails the guard `when is_binary(comment)` in `do_walk_assert_call_block`).